src/data/common/clients/storage/Namespaces.ts

```diff
@@ -9,6 +9,9 @@ export const Namespaces = {
     MAL_DIFF_STATUS_USER_COLUMNS: "mal-diff-status-user-columns",
     MAL_DIFF_NAMES_SORT_ORDER: "mal-diff-names-sort-order",
     MONITORING: "monitoring",
+    DATA_QUALITY: "data-quality-config",
```

Comment:
Minor: this isn't really used to store config, so this name ("data-quality-config") doesn't make sense. Maybe just "data-quality" or "data-quality-items"/"data-quality-entries"